axolotl-web/src/components/Message.vue

```diff
@@ -69,6 +69,10 @@
          </div>
        </div>
        <div class="message-text">
+         <blockquote v-if="message.quote != undefined">
```

Comment:
This should check for the message flags 14 
https://github.com/nanu-c/axolotl/blob/main/app/helpers/models.go#L25
And data structure looks like that 
![grafik](https://user-images.githubusercontent.com/404058/95864482-b9808e00-0d65-11eb-96c6-8d13d0c311b9.png)